CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+## How to contribute
+
+Please note that this project is released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating you agree to abide by its terms.
+
+We appreciate you taking the time to contribute to Octokit or any of the projects in Octokit's ecosystem. Especially as a new contributor, you have a valuable perspective: you will find things confusing and run into problems that no longer occur to the maintainers. Please share them with us so we can make the experience for future contributors the best it can be.
+
+Thank you 💖
+
+There are two types of contributions: issues and pull requests (PRs). Issues are used to track bugs and feature requests, while PRs contribute new content to the repository.
+
+## Creating an issue
+
+Before you create a new issue:
+1. Search the [project's issues](https://github.com/octokit/dotnet-sdk/issues) to see if a similar issue already exists. If so, please add onto that issue rather than creating a new one.
```

Comment:
```suggestion
1. Search the [project's issues](https://github.com/octokit/go-sdk/issues) to see if a similar issue already exists. If so, please add onto that issue rather than creating a new one.
```